scripts/order.json

```diff
@@ -39,8 +51,41 @@
             "401": { "type": "unit" },
             "404": { "type": "unit" }
           }
+        },
+        {
+        "method": "POST",
+          "body": {
+          	"type": "order_form"
+          },
+          "responses": {
+            "200": { "type": "order_form" },
+            "201": { "type": "order_form" },
+            "404": { "type": "unit" },
+            "422": { "type": "error" }
+          }
+        },
+        {
+        "method": "DELETE",
+        "responses":{
+             "200": { "type": "order" },
```

Comment:
We do not return an object when we delete it. So, this will be a unit.

Reply:
Okay